src/components/ProfileForm/components/ProfileForm/ProfileFormNavigationButtons.tsx

```diff
@@ -0,0 +1,44 @@
+import Button from "@/components/Button-Link/Button/Button";
+import {
+  faAngleDoubleLeft,
+  faAngleDoubleRight,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const ProfileFormNavigationButtons = ({
+  onPrevious,
+  onNext,
+}: {
+  onPrevious?: () => void;
+  onNext?: () => void;
+}) => {
+  return (
+    <div className="flex justify-around m-4">
+      {onPrevious !== undefined && (
+        <Button
+          variant="full"
+          color="black"
+          size="medium"
+          onClick={onPrevious}
+          className="rounded-md"
+        >
+          <FontAwesomeIcon icon={faAngleDoubleLeft} className="pr-1" />
+          Previous
+        </Button>
+      )}
+      {onNext !== undefined && (
+        <Button
+          variant="full"
+          color="black"
```

Comment:
Question: are all the form fields required, or no? Can you fill in only parts of them?

Asking this, because the answer might determine if we have a normal button here or a submit one